app/package.json

```diff
@@ -180,6 +180,7 @@
     "prop-types": "^15.6.2",
     "query-string": "^6.1.0",
     "raf": "^3.4.0",
+    "raw-loader": "^3.1.0",
```

Comment:
curiously, what's the difference between using `raw-loader` and `file-loader`?

Reply:
`raw-loader` gives you the file content directly as a string, so you don't have to `import`/`require`/`fetch` it separately.